src/pages/understand-stacks/integrate-stacking.md

```diff
@@ -326,8 +326,10 @@ Stacking delegation requires you to implement a different flow. Until the provid
 
 Below is a description of contract calls required to integrate delegation:
 
-1. Account holders call the [`delegate-stx`](/references/stacking-contract#delegate-stx) method and provide the delegator address, the amount accessible to the delegator, the address rewards should be send to, and the burn height at which the delegation relationship should be terminated
-2. For each delegation relationship that was created, the delegator calls the [`delegator-stack-stx`](/references/stacking-contract#delegate-stack-stx) method to lock up the STX token from the account holder. This method must be called until the delegator locked up enough STX tokens required to participate in Stacking
+1. Account holders call the [`delegate-stx`](/references/stacking-contract#delegate-stx) method and provide the delegator address, the amount accessible to the delegator, the address rewards should be send to, and the burn height at which the delegation relationship should be terminated. This needs to happen only once. The account holder can specify a delegation amount that is higher that their balance
```

Comment:
Maybe specify that when the locking is happening, the users should hold the expected amount?

Reply:
good point